apps/metricstream/template.yaml

```diff
@@ -61,11 +62,33 @@ Parameters:
     Description: |
       Buffer incoming data to the specified size, in MiBs, before delivering it
       to the destination.
-
+  ObserveAccountID:
```

Comment:
do we not need the cluster too? since the urls are different based on the cluster